File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {
    public int[] solution(String my_string) {
        int[] answer = new int[52];
        for(char c : my_string.toCharArray()){
            if('a' <= c && c <= 'z')
                answer[c - 'a' + 26]++;
            else
                answer[c - 'A']++;
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer(52);
    for(auto c : my_string){
        if('a' <= c && c <= 'z'){
            answer[c-'a'+26]++;
        } else{
            answer[c-'A']++;
        }
    }
    return answer;
}